src/main/java/com/codeborne/selenide/ex/TextsSizeMismatch.java

```diff
@@ -0,0 +1,22 @@
+package com.codeborne.selenide.ex;
+
+import com.codeborne.selenide.impl.WebElementsCollection;
+
+import java.util.List;
+
+public class TextsSizeMismatch extends UIAssertionError {
+  public TextsSizeMismatch(WebElementsCollection collection, List<String> actualTexts,
+                       List<String> expectedTexts, String explanation, long timeoutMs) {
+    super(collection.driver(),
+        "\nActual: " + actualTexts + ", List size: " + actualTexts.size() +
```

Comment:
Can you please use platform specific [System.lineSeparator()](https://docs.oracle.com/javase/8/docs/api/java/lang/System.html#lineSeparator--) insead of `\n` in exception class.

Reply:
fixed
``` super(collection.driver(),
        lineSeparator() + "Actual: " + actualTexts + ", List size: " + actualTexts.size() +
        lineSeparator() + "Expected: " + expectedTexts + ", List size: " + expectedTexts.size() +
        (explanation == null ? "" : lineSeparator() + "Because: " + explanation) +
        lineSeparator() + "Collection: " + collection.description());
```